pages/map.vue

```diff
@@ -88,12 +88,34 @@ const sortedDataByGrainType = computed(() => {
 
 valueColorScale.value = d3.scaleLinear().domain([0, 1000]).range(["white", "red"])
 
+// Merges geometries by shared ID.
+// Inspired by https://github.com/neocarto/geotoolbox/blob/cee58b6c45e3faa59ef680d8e3162c430077e80c/src/gis/aggregate.js
+const aggregate = (topology, objects, idProperty) => {
```

Comment:
nice and clean function love it. should we call it something like `mergeGeometriesByProperty`